.eslintrc

```diff
@@ -24,15 +24,15 @@
   ],
   "rules": {
     "@typescript-eslint/type-annotation-spacing": "error",
-    "no-unused-vars": "off", // note you must disable the base rule as it can report incorrect errors
+    "no-unused-vars": "off", // note: you must disable the base rule as it can report incorrect errors
     "@typescript-eslint/no-unused-vars": "error",
     "@typescript-eslint/consistent-type-assertions": "error",
     "no-array-constructor": "error",
     "@typescript-eslint/no-array-constructor": "error",
-    "no-empty-function": "error", // note you must disable the base rule as it can report incorrect errors
+    "no-empty-function": "off", // note: you must disable the base rule as it can report incorrect errors
```

Comment:
This check is helpful to have a better code base, keep it on. See https://eslint.org/docs/latest/rules/no-empty-function

Reply:
I just followed the suggestion from [here](https://typescript-eslint.io/rules/no-empty-function/#how-to-use) (since we're also using the `typescript-eslint` plugin).

Reply:
Ah, I see I missed the next line cause it was untouched. Good to go then 👍🏻